### ISSUE TITLE
How do I compile a parallel version of the opencv library

### ISSUE THREAD

**Issue by CodeCat00** [opencv/opencv#19350]
##### System information (version)

- OpenCV => 4.2
- Operating System / Platform => CentOS Linux release 7.6.1810
- Compiler => g++

##### Detailed description
I used the fusion function `seamlessClone` in my program to run 20 times slower on Linux than on windows, which I suspect is because the Linux version of the opencv library is not compiled into a parallel version. But I tried to add TBB compilation when compiling opencv, but there was still no parallel effect. How do I compile a parallel version of the opencv library, or how do I parallelize `seamlessClone`?

The version I use on the windows platform is windows 10, and the compiler is Visual Studio 2019, which is 4.2.0.

##### My code
```
Mat SplicingImage;
hconcat(inImage, inImage, SplicingImage);
vconcat(SplicingImage, SplicingImage, SplicingImage);

Mat mask = 255 * Mat::ones(inImage.rows, inImage.cols, inImage.depth());
Point centerPt = Point(SplicingImage.cols / 2, SplicingImage.rows / 2);
seamlessClone(inImage, SplicingImage, mask, centerPt, SplicingImage, NORMAL_CLONE);
// NORMAL_CLONE MIXED_CLONE MONOCHROME_TRANSFER

int cols = inImage.cols;
int rows = inImage.rows;
Rect rect = Rect(cols * (3. / 4), rows * (3. / 4), cols / 2, rows / 2);

resize(SplicingImage(rect), resImage, Size(510, 510));
```

##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [x] I checked the problem with documentation, FAQ, open issues,
       answers.opencv.org, Stack Overflow, etc and have not found solution
   
 -

**Comment by LaurentBerger** [comment]
@alalek may be this should be changed :
I checked the problem with documentation, FAQ, open **and close** issues,
answers.opencv.org, **forum.opencv.org**,  Stack Overflow, etc and have not found solution

**Comment by sturkmen72** [comment]
are you test it in release mode?

**Comment by CodeCat00** [comment]
> are you test it in release mode?

Yes, I use the default compilation mode of cmake, and the default compilation mode of cmake is release

**Comment by asmorkalov** [comment]
`seamlessClone` is pure serial function and does not contain parallel calls in implementation, besides may be some generic primitives. So the issue is not related to TBB, OpenMP or other threading framework. Please attach `cv::getBuildInformation()` call output for your code to check CMake options.
